book/preliminary/jupyterhub.md

```diff
@@ -1,86 +1,75 @@
-# JupyterHub Connection
+# JupyterHub 
 
 (event-jupyterhub)=
 ## Why are we using a shared cloud environment?
 
-Teaching software to a diverse group of participants, each with different computers and operating systems, can be challenging. As you'll learn in the next lesson, there are specific ways to configure our software for the tutorials to be successful, so it takes time to get everyone set up consistently. Our solution to this is to give everyone access to a cloud computing environment that is pre-configured for the specific software we will deploy. This cloud computing instance can be accessed from any web browser, which eliminates the need for configuring each person's individual computer.
+Teaching software to a diverse group of participants, each with different computers 
+and operating systems, can be challenging. As you'll learn in the next lesson, there 
+are specific ways to configure our software for the tutorials to be successful, so 
+it takes time to get everyone set up consistently. Our solution to this is to give 
+everyone access to a cloud computing environment that is pre-configured for the 
+specific software we will deploy. This cloud computing instance can be accessed 
+from any web browser, which eliminates the need for configuring each person's 
+individual computer.
```

Comment:
out of curiosity what editor are you using? seems like this is just changing max line length settings. something we could consider is some sort of pre-commit style enforcement, but that also adds complexity for people... so probably won't bother

Reply:
I use PyCharm and have a hard wrap guide at 80 and manually wrap it there. I also do this for any of my code as it increases readability. 
This could be done with a post-processor. Only concern I have there is that this sometimes can introduce errors, when the wrap is done automatically.